portfolio/src/main/webapp/script.js

```diff
@@ -36,22 +36,20 @@ async function addData() {
     document.getElementById('fetch-container').innerText = quote;
 }
 
+
 /**
- * Fetches stats from the servers and adds them to the DOM.
+ * Fetches the current state of the page and builds the UI.
  */
-function getServerData() {
-  fetch('/data').then(response => response.json()).then((stats) => {
-    // stats is an object, not a string, so we have to
-    // reference its fields to create HTML content
-
-    const statsListElement = document.getElementById('server-data-container');
-    statsListElement.innerHTML = '';
-    statsListElement.appendChild(
-        createListElement('First Message: ' + stats.firstMessage));
-    statsListElement.appendChild(
-        createListElement('Second Message: ' + stats.secondMessage));
-    statsListElement.appendChild(
-        createListElement('Third Message: ' + stats.thirdMessage));
+function getComment() {
```

Comment:
Since more than once comment can be saved, I'd suggest renaming this to getComments.